recipes/grpc/all/conanfile.py

```diff
@@ -99,11 +101,11 @@ def requirements(self):
         self.requires("grpc-proto/cci.20220627")
 
     def validate(self):
-        if self._is_msvc:
+        if is_msvc(self):
             if self.settings.compiler == "Visual Studio":
                 vs_ide_version = self.settings.compiler.version
             else:
-                vs_ide_version = tools.microsoft.visual.msvc_version_to_vs_ide_version(self.settings.compiler.version)
+                vs_ide_version = visual.msvc_version_to_vs_ide_version(self.settings.compiler.version)
```

Comment:
Not converted to new tools, can't be used as it's part of a private import. It would be great requesting it as a public module on https://github.com/conan-io/conan/issues

Reply:
https://github.com/conan-io/conan-center-index/issues/13555